spring-cloud-gcp-examples/spring-cloud-gcp-config-example/README.md

```diff
@@ -0,0 +1,83 @@
+# Spring Cloud Config Client for Google Cloud Runtime Configurator API
+
+This sample application demonstrates using the `Spring Cloud Config Client for GCP` in your code.
+
+## Setup
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/). You
```

Comment:
This isn't actually required, for example, if users use the core Boot starter and specify the credentials using `spring.cloud.gcp.credentials-location` property.